pkg/platform/aws/template.go

```diff
@@ -149,12 +149,13 @@ module "worker-pool-{{ $index }}" {
   {{- if $pool.SpotPrice }}
   spot_price            = "{{ $pool.SpotPrice }}"
   {{- end }}
-  {{- if $pool.TargetGroups }}
-  target_groups         = "{{ index (index $.WorkerpoolCfg $index) "target_groups" }}"
+
+	{{- if $pool.TargetGroups }}
```

Comment:
Indentation?

Reply:
Rendering was not done.

```{{- if ne (index (index $.WorkerpoolCfg $index) "clc_snippets") "null" }}``` was not correct as for a worker pool that didnt have clc_snippets. 

the rendered cluster.tf was `clc_snippets = `